src/CRM/CivixBundle/Resources/views/Code/entity-schema.xml.php

```diff
@@ -1,22 +1,22 @@
 <?php
 echo '<?xml version="1.0" encoding="iso-8859-1" ?>'."\n";
-$fixmeVersion = '4.4';
+$version = civicrm_api3('Extension', 'getvalue', ['key' => $mainFile, 'return' => 'version']);
```

Comment:
So I'm really unsure about this one. Random reactions:

* Is this just written as a WIP/gist, or does it actually work?
* The `<add>` / `<drop>` have traditionally been versions of `civicrm-core`.
* Does any process actually use this information right now? For example, scanning metadata? If so, are they more likely to break if they (a) continue to see core version#s or (b) see extension version#s?
* In the metadata for the entity, it would be useful to have some indication of origin.
* I usually try to avoid a booting/needing a running a Civi if I can avoid it, but it's not critical. I guess that `generate:entity` already. boots up to use`_civicrm_api_get_entity_name_from_camel()`.

CC @colemanw @eileenmcnaughton

Reply:
I don't think it actually does anything currently....